src/sentry/utils/assets.py

```diff
@@ -1,18 +1,31 @@
 from django.conf import settings
 
+import sentry
 
-def get_unversioned_asset_url(module, key):
+
+def get_unversioned_asset_url(module, key, cache_bust=False):
     """
     Returns an asset URL that is unversioned. These assets should have a
     `Cache-Control: max-age=0, must-revalidate` so that clients must validate with the origin
     server before using their locally cached asset.
 
+    XXX(epurkhiser): As a temporary workaround for flakeyness with the CDN,
+    we're busting caches when version_bust is True using a query parameter with
+    the currently deployed backend SHA. This will have to change in the future
+    for frontend only deploys.
+
     Example:
       {% unversioned_asset_url 'sentry' 'sentry.css' %}
       =>  "/_static/dist/sentry/sentry.css"
+
+      {% unversioned_asset_url 'sentry' 'sentry.css' cache_bust=True %}
+      =>  "/_static/dist/sentry/sentry.css?bust=xxx"
     """
+    args = (settings.STATIC_UNVERSIONED_URL.rstrip("/"), module, key.lstrip("/"))
 
-    return "{}/{}/{}".format(settings.STATIC_UNVERSIONED_URL.rstrip("/"), module, key.lstrip("/"))
+    if not cache_bust:
+        return "{}/{}/{}".format(*args)
+    return "{}/{}/{}?bust={}".format(*args, sentry.__build__)
```

Comment:
what about `v` or `ver`

```suggestion
    return "{}/{}/{}?ver={}".format(*args, sentry.__build__)
```

Reply:
👍